Source/WebCore/dom/Microtasks.h

```diff
@@ -22,12 +22,15 @@
 #pragma once
 
 #include "EventLoop.h"
+#include <JavaScriptCore/MicrotaskQueue.h>
```

Comment:
If we're including this header, then we don't need a forward declaration of MicrotaskQueue down below, right?

Reply:
Yes. Removed.